test/provider.ts

```diff
@@ -0,0 +1,9 @@
+const snapshot = require('../');
+
+const provider_DEFAULT = snapshot.utils.getProvider('1');
+console.log(provider_DEFAULT.connection.url == 'https://rpc.snapshot.org/1');
+
+snapshot.utils.setBroviderUrl('https://rpc.custom.org');
```

Comment:
You can actually use second param in getProvider to pass a different brovider URL, you tried?

Reply:
I tried. 
I'm using getVp and validate, but I want to use my own provider rpc. https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/utils/vp.ts#L22
https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/validations/basic/index.ts#L11

I use  snapshot-strategies package, don't direct add second param in getProvider.
So:
```
import strategies from '@snapshot-labs/strategies';
import snapshot from "@snapshot-labs/snapshot.js";

// change custom rpc
snapshot.utils.setBroviderUrl('https://rpc.custom.org');
// It use my custom rpc in getVp 
strategies.utils.getVp(...)
```